modules/rest/src/main/java/org/apache/ignite/internal/rest/recovery/system/SystemDisasterRecoveryController.java

```diff
@@ -61,7 +61,14 @@ public SystemDisasterRecoveryController(SystemDisasterRecoveryManager systemDisa
     public CompletableFuture<Void> reset(ResetClusterRequest command) {
         LOG.info("Reset command is {}", command);
 
-        return systemDisasterRecoveryManager.resetCluster(command.cmgNodeNames());
+        if (command.metastorageReplicationFactor() == null) {
```

Comment:
Let's introduce a method `metastorageRepairRequested()` to `ResetClusterRequest` and use it here